src/bikeracks/__init__.py

```diff
@@ -42,21 +49,34 @@ def create_app(test_config=None):
         pass
         
     # register close_db and init_db_command with the app
-    from . import db
     db.init_app(app)
     
     # import and register the blueprint from the factory
-    from . import bikes
-    from . import votes
     app.register_blueprint(bikes.bikes)
     app.register_blueprint(votes.votes)
         
     @app.route('/')
     def home(name=None):
         # update map with approved markers
-        # render_template invokes the jinja template engine
         return render_template('base.html', name=name)
         
+    @app.route('/submitFeedback', methods=('POST',))
+    def submitFeedback():
+        csv_file = os.path.join(app.instance_path, "feedback.csv")
+        timestamp = int(time())
+        feedback = request.form.get('feedback', type=str)
```

Comment:
You probably want to be a little more cautious with this. I uploaded a bunch of trash, for example.

Reply:
Encode with utf-8. (Am I saying that right?)

Reply:
"encode _as_utf-8" is how people normally refer to it. Flask is apparently decoding already to give you a unicode `str` from `request.form.get()`.

I'm thinking more about the length.

Reply:
But doing it explicitly anyway too... the encoding I mean

Reply:
You can't. It's decoded from a 'utf-8' bytestring to unicode before you ever get the object. You are free to reencode it as 'utf-8' when writing, but you do that already by opening the file with `encoding='utf-8'`